sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -114,15 +111,16 @@ export const ContentPlacementContainerStyles = css`
     @media screen and (max-width: 1330px) {
         :host([section="feature"]) {
             --flow: row;
+            grid-template-columns: unset;
         }
         site-feature-card:hover + site-feature-card::before {
             opacity: 0;
         }
     }
 
-    @media screen and (max-width: 750px) {
+    @media screen and (min-width: 750px) {
```

Comment:
should we change this to 900px so that the change of layout for community lines up with the change of layout for feature and the headings switching to their more condensed layout?

Reply:
I'm happy to look at it and see in the morning, although I believe this is something that we'd want input from Chris and Jason on.

Reply:
Looked at this and setting 900 for the community cards isn't really a good look.  They look stretched.  But that's just my opinion.

Reply:
could we set them to have the same width then as the features card above them to avoid them looking stretched?

Reply:
Should we separate these visual changes into another PR to unblock the rest of this?

Reply:
Being that the focus of this PR was about the links and text in the feature cards, I think changes to the Community section appearance should probably be addressed in a separate issue/PR.  The minor change I put in here was before the footer overlap fix went in and it made things at the bottom of the page change at the same point.  Now that the footer overlap is fixed, that minor code change could come out and the bottom looks basically the same as it does with it there.